docs/index.md

```diff
@@ -0,0 +1,104 @@
+---
+# https://vitepress.dev/reference/default-theme-home-page
+layout: home
+
+hero:
+  name: "PGlite"
+  text: "Lightweight WASM Postgres everywhere"
```

Comment:
Just a few thoughts on copy.

I think you can simplify the headline and expanding the subline, e.g.:

```
PGlite
WASM Postgres
Lightweight embeddable Postgres from ElectricSQL
```

Or if WASM is too specific, given the goals to have a lib, perhaps:

```
PGlite
Embeddable Postgres
Lightweight WASM Postgres from ElectricSQL
```

You could also make it more general language:

```
PGlite
Postgres in the browser
Lightweight WASM Postgres for web, edge and Node/Bun/Deno.
```